src/client/ui/selector-inspector-panel/hide-button.js

```diff
@@ -0,0 +1,38 @@
+import hammerhead from './../deps/hammerhead';
+import testCafeCore from './../deps/testcafe-core';
+import { createElementFromDescriptor } from './utils/create-element-from-descriptor';
+import { setStyles } from './utils/set-styles';
+
+import * as descriptors from './descriptors';
+
+const nativeMethods = hammerhead.nativeMethods;
+
+const eventUtils = testCafeCore.eventUtils;
```

Comment:
Let keep the `const` statements together without extra lines between them.
However, line after the `import` statement is ok